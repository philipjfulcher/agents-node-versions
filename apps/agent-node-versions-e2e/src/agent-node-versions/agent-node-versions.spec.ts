import { execSync } from 'child_process';
import { join } from 'path';

describe('CLI tests', () => {
  it('should print a message', () => {
    const cliPath = join(process.cwd(), 'dist/apps/agent-node-versions');

    const output = execSync(`node ${cliPath}`).toString();

    expect(output).toMatchSnapshot();
  });
});
