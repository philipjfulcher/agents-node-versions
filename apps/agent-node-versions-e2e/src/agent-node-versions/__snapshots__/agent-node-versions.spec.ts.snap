// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CLI tests should print a message 1`] = `
"Hello World
[33mfalse[39m
[33mfalse[39m
[33mfalse[39m
[33mfalse[39m
[33mtrue[39m
[33mtrue[39m
"
`;
